suite-common/wallet-core/src/send/sendFormTypes.ts

```diff
@@ -53,6 +53,7 @@ export type ComposeFeeLevelsError = {
 
 export type SignTransactionError = {
     error: 'sign-transaction-failed';
+    connectErrorCode?: string;
```

Comment:
nitpick: I would prefer just `errorCode`

Reply:
fixed in faf9766feb

Also Connect typing improved in [a631499](https://github.com/trezor/trezor-suite/pull/14609/commits/a63149905359c4094e0488aefc48a103ae318095). @mroz22 Could you please look at it? 🙏